src/stirling/e2e_tests/stirling_wrapper_container_bpf_test.sh

```diff
@@ -48,7 +48,7 @@ run_uprobe_target "$go_grpc_server" "$go_grpc_client"
 echo "Running stirling_wrapper container."
 
 flags="--timeout_secs=0"
-out=$(podman run --init --rm \
```

Comment:
What error did you specifically see? Curious to know how it manifested.

Reply:
> What error did you specifically see? Curious to know how it manifested.

Error: container-init binary not found on the host: stat /usr/libexec/podman/catatonit: no such file or directory^M

We can probably add that to our sysroot, but it will just be bloat.